common-react/@dbeaver/ui-kit/src/Checkbox/Checkbox.css

```diff
@@ -0,0 +1,98 @@
+@import './_base.css';
+@import './_sizes.css';
+
+@layer base {
```

Comment:
Should we use the `components` layer instead?

Reply:
base is good